vita3k/touch/src/touch.cpp

```diff
@@ -189,6 +211,57 @@ int handle_touch_event(SDL_TouchFingerEvent &finger) {
     return 0;
 }
 
+int handle_touchpad_event(SDL_ControllerTouchpadEvent &touchpad) {
+    switch (touchpad.type) {
+    case SDL_CONTROLLERTOUCHPADDOWN:
+        if (finger_count >= 2) // best we can do is clean everything
```

Comment:
Is there a reason as to why only 2 fingers at most are supported. On its touchscreen the PSVita supports up to 8 fingers simultaneously, and SDL does not look like it has a limit to the number of supported fingers.

Reply:
For touchpad of ds/4 is 2 maximum for that, no i have thinks no need set more 🤷‍♂️

Reply:
No idea have have really interest but so i have set also 8 for it